### PR TITLE
Allows popover to receive a spacing for positioning from the top

### PR DESCRIPTION
**Pull request by mfrachet**
## Description

Allows popover to receive a spacing for positioning from the top

### CODE REVIEW

File: packages/strapi-design-system/src/Popover/Popover.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ const PopoverScrollable = styled(Box)`
   }
 `;
 
-export const Popover = ({ source, visible, children, ...props }) => {
+export const Popover = ({ source, visible, children, spacing, ...props }) => {
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Popover/Popover.js:48]: That spacing prop can be a bit confusing. As a developer, I expect to have spacing on every side, not just the top.

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-design-system/src/Popover/Popover.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-design-system/src/Popover/Popover.js:48]: Sounds good 😊 what could be another valid name?

Comment (HichamELBSI) [reply packages/strapi-design-system/src/Popover/Popover.js:48]: I don't really know maybe something like `spacingTop`, `verticalSpacing` or `marginTop`.

Comment (mfrachet) [reply packages/strapi-design-system/src/Popover/Popover.js]: I've taken spacingTop

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**